spec/controllers/rentals_controller_spec.rb

```diff
@@ -47,11 +48,46 @@
   end
 
   describe 'GET #cost' do
-    it 'returns a cost based on item type' do
-      start_time = Date.today
-      end_time = Date.tomorrow
-      get :cost, params: { item_type: item_type, start_time: start_time, end_time: end_time }
-      expect(response.body).to eq(Rental.cost(start_time, end_time, item_type).to_s)
+    context 'retrieves cost' do
```

Comment:
A lot of repetition in these tests. Up to you as a team to decide whether you prefer that or prefer some abstraction with `let` statements and the like.